include/vcpkg/base/messages.h

```diff
@@ -670,6 +672,15 @@ namespace vcpkg
                     "Fourth optional part of build troubleshooting message, printed after the version"
                     "information about vcpkg itself.",
                     "Please use the prefilled template from {path} when reporting your issue.");
+    DECLARE_MESSAGE(
+        BuiltinBaseHelp,
+        (),
+        "",
+        "The baseline references a commit within the vcpkg repository that establishes a minimum version on "
+        "every dependency in the graph. For example, if no other constraints are specified (directly or "
+        "transitively), then the version will resolve to the baseline of the top level manifest. Baselines "
+        "of transitive dependencies are ignored.");
+    DECLARE_MESSAGE(BuiltInTriplets, (), "", "VCPKG built-in triplets:");
```

Comment:
I remember that @ AugP opened a PR in the vcpkg repo to lowercase all occurrences of `vcpkg` in the documentation. We might want to do this here as well?

Reply:
Interesting, I will have to ask him :)